src/main/java/com/plaid/client/request/LinkTokenCreateRequest.java

```diff
@@ -0,0 +1,172 @@
+package com.plaid.client.request;
+
+import com.google.gson.annotations.JsonAdapter;
+import com.plaid.client.internal.gson.Optional;
+import com.plaid.client.internal.gson.OptionalDateTypeAdapterFactory;
+import com.plaid.client.request.common.BaseClientRequest;
+import java.util.Date;
+import java.util.List;
+import java.util.Map;
+
+public class LinkTokenCreateRequest extends BaseClientRequest {
+  private User user;
+  private String clientName;
+  private List<String> products;
+  private List<String> countryCodes;
+  private String language;
+  private String webhook;
+  private String accessToken;
+  private String linkCustomizationName;
+  private String redirectUri;
+  private String androidPackageName;
+  private Map<String, SubtypeFilters> accountFilters;
+  private LinkTokenCreateRequest.CrossAppItemAdd crossAppItemAdd;
+  private String paymentInitiation;
+
+  public LinkTokenCreateRequest(
+    User user,
+    String clientName,
+    List<String> products
+  ) {
+    this.user = user;
+    this.clientName = clientName;
+    this.products = products;
+  }
+
+  public static class SubtypeFilters {
+    private List<String> accountSubtypes;
+    private List<String> subtypes;
+
+    public SubtypeFilters(List<String> accountSubtypes) {
+      this.accountSubtypes = accountSubtypes;
+    }
+  }
+
+  public static class User {
+    private String clientUserId;
+    private String legalName;
+    private String phoneNumber;
+    private String emailAddress;
+    @JsonAdapter(OptionalDateTypeAdapterFactory.class)
+    private Optional<Date> phoneNumberVerifiedTime = Optional.empty();
+    @JsonAdapter(OptionalDateTypeAdapterFactory.class)
+    private Optional<Date> emailAddressVerifiedTime = Optional.empty();
+
+    public User(String clientUserId) {
+      this.clientUserId = clientUserId;
+    }
+
+    public User withClientUserId(String clientUserId) {
```

Comment:
Wondering if we should either only include this in the User constructor or have this 🤔